automatic/wps-office-free/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,22 @@
+
+  $version = '10.2.0.5820'
+  $regName = $packageName -replace('\-',' ')
```

Comment:
I will change this, but (FYI) a Chocolatey ( side effect ) is `$packageName` already defined by Chocolatey before this file is even executed.